maintainers/maintainer-list.nix

```diff
@@ -3426,6 +3426,12 @@
     githubId = 62989;
     name = "Demyan Rogozhin";
   };
+  dennajort = {
+    email = "1536838+dennajort@users.noreply.github.com";
```

Comment:
Would you mind if you used an email not github.com? Otherwise it could be hard to reach you in the future.